app/deployment_create.py

```diff
@@ -148,12 +168,18 @@ def init(ctx, output):
         spec_file_content.update(default_spec_file_content)
     if verbose:
         print(f"Creating spec file for stack: {stack}")
+
+    ports = _get_named_volumes(stack)
```

Comment:
Something looks wrong about this line.

Reply:
Hah!  That's what I get for making a last minute tweak.